guide/src/types.md

```diff
@@ -295,7 +295,7 @@ usually defined using the `#[pyclass]` macro.
 This trait marks structs that mirror native Python types, such as `PyList`.
 
 
-[eval]: {{#PYO3_DOCS_URL}}/pyo3/struct.Python.html#method.eval
```

Comment:
preview: https://deploy-preview-3481--pyo3.netlify.app/main/types#the-python-gil-mutability-and-rust-types